html/browsers/the-window-object/window-indexed-access-vs-named-access.html

```diff
@@ -0,0 +1,49 @@
+<!doctype html>
+<meta charset=utf-8>
+<title>Interactions between indexed and named access on the Window object</title>
+<link rel="author" title="Delan Azabani" href="dazabani@igalia.com">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/#accessing-other-browsing-contexts">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/#named-access-on-the-window-object">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div id=0></div>
+<div id=3></div>
+<iframe name=2></iframe>
+<iframe name=1></iframe>
```

Comment:
We miss a case where the name number is less than the index.

Reply:
done in 79121d3812c3ae2d3629dd36d3f09d8173d03158